render-configs.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
For the main jade-data-repo, we do not have the deployment and vault stuff in the github repository. It is in a private repo. I can't see that it matters to have this file in the project, though.

Perhaps add a comment that this is specific to the Broad deployment?

@thathert, what do you think?

Reply:
Yes, that's a good point. We should note the intention of this script is to fetch secrets from vault, but that it is Broad-specific.

Reply:
Done. Added a comment that it is specific to the Broad deployment.

Also note that a very similar script is checked into the main Data Repo GitHub repository.